x/group/migrations/v2/migrate_test.go

```diff
@@ -0,0 +1,81 @@
+package v2_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/cosmos/cosmos-sdk/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	authkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/group"
+	"github.com/cosmos/cosmos-sdk/x/group/internal/orm"
+	groupkeeper "github.com/cosmos/cosmos-sdk/x/group/keeper"
+	v2 "github.com/cosmos/cosmos-sdk/x/group/migrations/v2"
+	groupmodule "github.com/cosmos/cosmos-sdk/x/group/module"
+)
+
+var (
+	policyAddr  = sdk.MustAccAddressFromBech32("cosmos1q32tjg5qm3n9fj8wjgpd7gl98prefntrckjkyvh8tntp7q33zj0s5tkjrk")
+	accountAddr = sdk.AccAddress("addr2_______________")
+)
+
+func TestMigrate(t *testing.T) {
+	cdc := moduletestutil.MakeTestEncodingConfig(auth.AppModuleBasic{}, groupmodule.AppModuleBasic{}).Codec
+	storeKey := sdk.NewKVStoreKey(v2.ModuleName)
+	tKey := sdk.NewTransientStoreKey("transient_test")
```

Comment:
why transient?

Reply:
Because it is what `testutil.DefaultContext` expects.